client/app/actions/commentsActionCreators.js

```diff
@@ -43,24 +45,34 @@ export function submitCommentFailure(error) {
 
 export function fetchComments() {
   return dispatch => {
+    dispatch(setIsFetching());
     return (
-      commentsManager
-        .fetchComments()
-        .then(res => dispatch(fetchCommentsSuccess(res.data)))
-        .catch(res => dispatch(fetchCommentsFailure(res.data)))
+        commentsManager
```

Comment:
why the extra indentation?

In general, try to limit whitespace diffs unless the linter is complaining.

Maybe the linter wants extra indentation?